app/assets/javascripts/manage_submissions.js

```diff
@@ -44,14 +44,20 @@ $(document).ready(function () {
 
     // Fetch data and render it in the modal 
     get_score_details(course_user_datum_id).then((data) => {
+      const sorting_icons =
```

Comment:
![Screenshot 2023-10-06 at 02 29 08](https://github.com/autolab/Autolab/assets/9074856/cc3c996f-bb71-4150-8f2a-8d7b9f4dcfa0)

Note that once datatables-rows is correctly included, there are existing sorting icons

Reply:
True, but I personally think we should remove the link to Datatables stylesheet and use the styling/logic Lynn implemented because:
1) The default Datatables styling results in the icon appearing on all columns and sorting a column like "Actions" is weird. Disabling the icons is not straightforward: https://datatables.net/forums/discussion/41214/how-to-leave-column-sorting-enabled-but-remove-the-sorting-icons
2) The icons that Lynn added looks better IMO and changing the original icons to use the materialize icons would require the same amount of code.

Reply:
Agree with Kester